pipeline-scripts/commonlib.groovy

```diff
@@ -499,17 +499,17 @@ def canLock(lockName, timeout_seconds=10) {
  * on different routes. This method returns a URL based on the name of the
  * release stream you want to query.
  * @param releaseStreamName - e.g. 4-stable or 4-stable-s390x
- * @return Returns something like "https://openshift-release-s390x.svc.ci.openshift.org"
+ * @return Returns something like "https://s390x.ocp.releases.ci.openshift.org"
  */
 def getReleaseControllerURL(releaseStreamName) {
-    def archSuffix = ''
+    def archSuffix = 'amd64'
     def streamNameComponents = releaseStreamName.split('-') // e.g. ['4', 'stable', 's390x']  or [ '4', 'stable' ]
     if ('s390x' in streamNameComponents) {
-        archSuffix = "-s390x" // e.g. -s390x
+        archSuffix = "s390x" // e.g. -s390x
     } else if ('ppc64le' in streamNameComponents) {
-        archSuffix = "-ppc64le"
+        archSuffix = "ppc64le"
     }
-    return "https://openshift-release${archSuffix}.svc.ci.openshift.org"
+    return "https://${archSuffix}.ocp.releases.ci.openshift.org"
```

Comment:
This will seem nit-picky but... it's no longer a suffix. can the variable just be named `arch`?

Reply:
updated: https://github.com/openshift/aos-cd-jobs/pull/2406/commits/6d8cd6cadd0082f031d3f6567cdcf0799f714d07